common/crypto/attestation-api/test/conversion_app_go/main.go

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Obsolete?

Reply:
Per conversations in the PDO community meeting, I have removed the dead Go code.